cineast-core/src/main/java/org/vitrivr/cineast/core/color/RGBContainer.java

```diff
@@ -110,4 +110,16 @@ public RGBContainer set(int intcolor) {
     return this;
   }
 
+  public int getR() {
```

Comment:
I think the getters should belong in the `ReadableRGBContainer` abstraction layer